### DISCUSSION TITLE
Commerce Integration, Actions

### DISCUSSION THREAD

**Discussion by acalvino4** [trendyminds/craft-palette#7] [Ideas]
Great, smooth plugin!

Just a few areas where searching could be improved to bring what I expect:
* With Commerce installed, searching Product, Order, etc. doesn't bring up any option for the commerce subtabs
* Going directly to the creation of an entry, category, etc.
* Same thing for edit.

I love Craft way better than Expression Engine, but I have to say, their new jump menu is pretty slick... https://www.youtube.com/watch?time_continue=27&v=DqhPSPHPLc4&feature=emb_logo

Anything from that which could be integrated here would be slick.

No idea how much work all that is, so thanks for the plugin and thanks for any openness to improvements!

**Comment by aaronbushnell** [comment]
Thanks, @acalvino4! We don’t personally have any Commerce sites, but we’ll welcome a PR from anyone who does and wants to contribute those actions into the codebase.
